tests/api/testcl.hpp

```diff
@@ -760,5 +761,23 @@ class WithCommandQueue : public WithContext {
 
 class WithProfiledCommandQueue : public WithCommandQueue {
 protected:
-    void SetUp() override { SetUpQueue(CL_QUEUE_PROFILING_ENABLE); }
+    void SetUp() override {
+        WithContext::SetUp();
+        SetUpQueue(CL_QUEUE_PROFILING_ENABLE);
+    }
+};
+
+class WithPrintfEnabled : public WithCommandQueue {
+protected:
+    void SetUp() override{};
+    void TearDown() override {
+        if (m_queue != nullptr) {
```

Comment:
Should not be needed, if there is a test to do, it should be inside `WithCommandQueue::TearDown`.